style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-image: url(./assets/geo.png);
+  background-repeat: no-repeat;
+}
+
+.navbar {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+}
+
+.tri-svg {
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 70px;
+  width: 70px;
+}
+
+.sq-svg {
+  position: absolute;
+  left: 32px;
+  top: 0;
+  height: 70px;
+  width: 70px;
+}
+
+.menu {
+  margin-top: 30px;
+  margin-right: 16px;
+}
+
+.hero-section {
+  display: flex;
+  justify-content: start;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.heading {
+  font-family: 'Playfair Display', serif;
+  color: #1a2236;
+  font-size: 50px;
+  margin-left: 16px;
+  margin-top: 104px;
+  line-height: 50px;
+  margin-bottom: 16px;
+  text-align: left;
+}
+
+.para {
+  font-family: 'Poppins', sans-serif;
+  padding: 0 16px;
+}
+
+.btn-section {
+  display: flex;
+  justify-content: center;
+}
+
+.btn {
```

Comment:
[ ] As it's mentioned by the previous reviewer, kindly add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:🤓 
<img width="279" alt="hover" src="https://user-images.githubusercontent.com/62156607/198245827-a7cf20c8-97ba-4407-a5fd-64f0ff4fa151.png">